File: java examples/Arcade Drive/java/org/team1085/guppy/Robot.java
package org.team1085.guppy;

import edu.wpi.first.wpilibj.IterativeRobot;
import edu.wpi.first.wpilibj.command.Scheduler;
import org.team1085.guppy.subsystems.Chassis;
import org.team1085.guppy.subsystems.Dashboard;

public class Robot extends IterativeRobot {

  public static Chassis chassis = new Chassis();
  public static Dashboard dashboard = new Dashboard();

  public void robotInit () {}

  public void autonomousInit () { Scheduler.getInstance().removeAll(); }
  public void teleopInit () { Scheduler.getInstance().removeAll(); }

  public void autonomousPeriodic () { Scheduler.getInstance().run(); }
  public void teleopPeriodic () { Scheduler.getInstance().run(); }

}

File: kotlin exampes/Arcade Drive/kotlin/org/team1085/guppy/Robot.kt
package org.team1085.guppy

import edu.wpi.first.wpilibj.IterativeRobot
import edu.wpi.first.wpilibj.command.Scheduler
import org.team1085.guppy.subsystems.Chassis
import org.team1085.guppy.subsystems.Dashboard

class Robot : IterativeRobot () {

  companion object {
    public val chassis = Chassis()
    public val dashboard = Dashboard()
  }

  override fun robotInit () {}

  override fun autonomousInit () = Scheduler.getInstance().removeAll()
  override fun teleopInit () = Scheduler.getInstance().removeAll()

  override fun autonomousPeriodic () = Scheduler.getInstance().run()
  override fun teleopPeriodic () = Scheduler.getInstance().run()

}